llvm/test/Analysis/LoopAccessAnalysis/print-order.ll

```diff
@@ -0,0 +1,65 @@
+; REQUIRES: asserts
+; RUN: opt -passes='loop-vectorize' -debug-only=loop-accesses -disable-output < %s 2>&1 | FileCheck %s
```

Comment:
If placed in `llvm/test/Analysis/LoopAccessAnalysis/`, this should use `-passes='print<access-info>'` if possible